src/Quartz/DateBuilder.cs

```diff
@@ -104,7 +92,7 @@ private DateBuilder(TimeZoneInfo tz)
     /// <returns></returns>
     public static DateBuilder NewDate()
```

Comment:
maybe needs parameter `TimeProvider? timeProvider = null` and then we default to system?